src/implementations/twig/components/page-header/ec-page-header-core.story.js

```diff
@@ -0,0 +1,182 @@
+import { withNotes } from '@ecl/storybook-addon-notes';
+import withCode from '@ecl/storybook-addon-code';
+import { correctSvgPath } from '@ecl/story-utils';
+
+import demoContent from '@ecl/specs-component-page-header/demo/data--default';
+import demoBackgroundImage from '@ecl/specs-component-page-header/demo/data--background-image';
+import demoBreadcrumbLongEC from '@ecl/specs-component-breadcrumb/demo/data--ec';
+
+import pageHeader from './page-header.html.twig';
+import notes from './README.md';
+
+const dataDefault = { ...demoContent };
+const dataBackgroundImage = { ...demoBackgroundImage };
+
+const getArgs = (data) => {
+  const args = {
+    breadcrumb: true,
+    thumbnail: false,
+  };
+
+  if (data.title) {
+    args.title = data.title;
+  }
+  if (data.meta) {
+    args.meta = data.meta;
+  }
+  if (data.background_image_url) {
+    args.overlay = 'none';
+  }
+  if (data.description) {
+    args.description = data.description;
+  }
+  if (data.background_image_url) {
+    args.background_image_url = data.background_image_url;
+  }
+
+  return args;
+};
+
+const getArgTypes = (data) => {
+  const argTypes = {};
+
+  argTypes.breadcrumb = {
+    name: 'breadcrumb',
```

Comment:
we should keep the same kind of structure/naming for the optional controls:
show_breadcrumb, and put in an "optional" group
(check the content-item for instance)